src/db/__init__.py

```diff
@@ -246,6 +246,11 @@ def get_completed_survey_channel_ids_older_than(minus_interval):
     return [res[0] for res in _fetchall(query)]
 
 
+def get_in_progress_survey_channel_ids_created_older_than(minus_interval):
```

Comment:
Is the `created` part important in the name? I would rather name it `get_in_progress_survey_channel_ids_older_than`. As is actually done with `get_completed_survey_channel_ids_older_than`.

Reply:
Yeah ok, I agree